### PR TITLE
ch03s02 practices inspect ge

### PR DESCRIPTION
**Pull request by gjbianco**

### CODE REVIEW

File: models/practices-inspect/pandas-stress.ipynb [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    "df = pd.read_csv('./rand_people.csv', index_col=['Initials'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57343de0-e067-4135-9b89-3fad06fd9d39",
+   "metadata": {},
+   "source": [
+    "3. From the right-side menu, open the `Kernel Usage` panel.\n",
+    "Observe that the Jupyter kernel CPU and memory are less than ten percent and under one-hundred megabytes, respectively."
```
Comment (jramcast) [inline models/practices-inspect/pandas-stress.ipynb:50]: At this step I have slightly a bit more than 100 MB. 

image

We can just tell students to verify that the memory is under 200.

File: models/practices-inspect/pandas-stress.ipynb [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    "5. As the code in the preceding cell runs, observe the CPU and memory usage from the `Kernel Usage` panel in the right side menu.\n",
+    "In particular, the CPU usage is around 100% and the memory usage grows to several gigabytes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64e04b36-63f6-45ba-8c71-bebe66c3aed6",
+   "metadata": {},
+   "source": [
+    "6. Within the same panel, under the `Host CPU` section, observe that the CPU percentage is around 10% to 15% usage.\n",
```
Comment (jramcast) [inline models/practices-inspect/pandas-stress.ipynb:92]: I had a bit more than this range:

image


```suggestion
    "6. Within the same panel, under the `Host CPU` section, observe that the CPU percentage is around 10% to 25% usage.\n",
```

**Review by jramcast — APPROVED** [review]
LGTM. Just a few comments about the expected memory and CPU utilization ranges.

**Review by sffrench — APPROVED** [review]
LGTM

**Pull request merged**